### DISCUSSION TITLE
OpenRadioss with One Step Stamping

### DISCUSSION THREAD

**Discussion by YongCodeRadioss** [OpenRadioss/OpenRadioss#1166] [Users]
Does OpenRadioss support one step stamping simulation? I have tried to run the same output file from HyperForm without success. I used -onestep or regular Starter/Engine. Any suggestions?

**Comment by MarianBulla** [comment]
Hello,
Thank you very much for your valuable question. 
Currently, OpenRadioss does not support one-step-stamping, since it is an implicite solution.
It is embedded in HyperMesh, where a simplified solution is executed to quickly calculate the results.
Technically, it should be possible to add it into OpenRadioss. Maybe someone with some knowledge can implement it, from the community?
Thanks and best regards,
Marian
